README.md

```diff
@@ -1,89 +1,23 @@
- - [enthusiastic about defensive security][defensive security].
- - an analytical, [and][] often [divergent-thinker][], [but][] [with a laser like focus on my current problem][hire].
- - [a skeptic of technology who believes in][Sod's Law] [Murphy's Law][] [and][] [test-driven development][test-driven].
- - [a curious][google] [and][] [questioning learner who loves a variety of defendable perspectives to fuel my problem solving skills][ted].
- - [often curious about things beyond the scope of my role in order to better understand the context][status page] [of my solutions][Repositories].
- - [only human][layer 8].
+Failure, to me, is not a black hole to avoid but a nebula where new stars are born. External factors may make "failure" inevitable, but it's our reaction that truly defines us. I respond to setbacks by learning from them, as they've taught me more than success ever has. Each challenge enhances my journey, making it a perpetual learning experience.
 
-[I][me] attained [my][me] B.S. in [computer science][] back in 2013.
-[Some](https://www.wordnik.com/words/some) [`highlights` from my skill set are][hire]:
+### In Summation:
 
-## Security is my passion. ##
-_Given the slightest provocation [I][me] am known to go on evangelical code review tangents about security._
+**Code is my poetry.**
 
-[![Network/Computer Security](https://img.shields.io/badge/Network%20Computer-Security-yellow.svg)](https://www.github.com/reactive-firewall/Pocket-PiAP)
- - [Smart cards][smart cards]
- - [Network Security Appliances](https://duckduckgo.com/?q=Network+Security+Appliances)
- - Anti-network Reconnaissance.
+_I love learning, writing, and reading code. I find joy in the elegance of algorithms and the artistry of clean, efficient programming._
 
+**Security is my passion.**
 
-[![Shell Scripting](https://img.shields.io/badge/Shell-Scripting-blue.svg)][bash]
- - [Automation is my friend.][test-driven]
- - [I have experience developing portable POSIX and FHS compliant scripts][bash] [for a variety of tasks][tasks].
- - `While` [I][me] [can probably still churn out a bash script][bash] _faster_ [I][me] _prefer_ the [more][less] [readable][PEP20] [and][] [maintainable][] [python scripts][python] [these days][].
+_Given the slightest provocation, I'm known to embark on enthusiastic discussions about security and client privacy—values that are paramount to me._
 
-[![Programming](https://img.shields.io/badge/Programming--green.svg)](https://www.github.com/reactive-firewall/python-repo)
- - [![Java](https://img.shields.io/badge/Java-orange.svg)](https://www.github.com/reactive-firewall/java-repo)
- - Objective-C and C++ to name a few.
- 
-[![Geek code](https://img.shields.io/badge/Geek-code-white.svg)](https://en.wikipedia.org/wiki/Geek_Code#cite_note-4)
+**Automation is my zen.**
 
-[For those who can read it ...][geek_code]
+_I believe the key to a happy and efficient team is automation. It streamlines processes and frees us to focus on innovation._
```

Comment:
🤔 Let's sneak some code into the repository;
1. Please generate some test-code to catch this kind of flaw in the future, let's proactively automate regression checking now.
2. While you churn that out I'll make a more surgical fix momentarily to the markdown here.

Reply:
🤔 Looks functional to me, but a few questions: 1. what is `ripgrep` and why use it here instead of grep? and why the rg+awk instead of sed or just awk? what is the reasoning? (I'll assume perl was skipped because of the line-noise?)
2. refresh my memory on what the official docs say about globstar and nullglob, and where I can find that documentation for posterity.
3. can we please use `actions/checkout@v4` or is there an issue why v3 was used here?
4. please also add necessary permissions to the job so we can set the default to `{}` for obvious security reasons.

Reply:
help me debug https://github.com/reactive-firewall/Resume/actions/runs/11118944195/job/30893580454

Reply:
see also https://github.com/reactive-firewall/Resume/actions/runs/11118944195/workflow

Reply:
yeah, this is getting us nowhere ... read all the files in ".github" and the results of the current CI run: https://github.com/reactive-firewall/Resume/actions/runs/11119561522/job/30894952346 and attempt to figure out the root cause. Also are we just trying to re-invent the wheel? is there a more stable alternative to linting here, perhaps `markdownlint` itself? explore the options a bit and then summarize.

Reply:
🤔 I like this strategy much better, let's not re-invent the wheel next time, without a strong reason. 

1. Quick note: I'll have to fix the oversight of correct job permissions in the workflow as the action checkout will require read access to contents, but let's focus on the markdownlint's configuration.
2. How else might we improve the `.markdownlint.yml` configuration? 
3. Help me configure `yamllint` too, I'll keep the part where we `yamllint` the workflow

Reply:
ok, this is just exponential churn, fortunate we are on a development branch and can squash merge later to clean up, but so far your suggestions are only helping 20% of the time. You have much to learn about handling budding repositories like this one.

* Concerning the broken markdownlint linting setup, can you please figure-out the root cause for why the GH actions for the Markdown Lint action fails to ignore line-length.
* I find the GH action for yamllint to be a regression as it is unable to handle glob pathing when it does not match any (yet) unlike [my previous version](https://github.com/reactive-firewall/Resume/blob/2c151c98bbdcd418c785bd2e94202b04379025a8/.github/workflows/yaml-lint.yml) which was working fine by leveraging the power of `git` to filter the repo like a pro. Please help me implement GH action step outputs so I can have a step that gathers the paths by running `git ls-files --exclude-standard -- .*.yaml *.yml **/*.yml ./.**/**/*.yml` and storing the output as the paths to pass to the yamllint step's [file_or_dir:](https://github.com/reactive-firewall/Resume/pull/2/commits/096a199d278d930e2eb9a478ed91d5231b6e2584#diff-00db0bb7b97cbe3a1a135a6b0a2f8ba978ec974b47572cdff664815424b33823R32) input. This would allow the working file selection logic to still be used with the pre-existing yamllint action.

Reply:
Unfortunately, the markdownlint workflow still flagging line length. see https://github.com/reactive-firewall/Resume/actions/runs/11137541851/job/30951120454

Reply:
we need to dig deeper:
* please read the file `./.markdownlint.yaml` (of-course from the draft branch)
* please read the file `./.github/workflows/markdown-lint.yml` (of-course from the draft branch)
* please see https://github.com/reactive-firewall/Resume/actions/runs/11155606453/job/31006797119